docs/getting-started/integrate/node.mdx

```diff
@@ -67,34 +66,34 @@ afterAll(() => server.close())
 
 ### Using manual setup
 
-If you are not using CRA, please create a custom teardown module and reference it using Jest's [`setupFilesAfterEnv`](https://jestjs.io/docs/en/configuration#setupfilesafterenv-array) option in your `jest.config.js`.
+If you are not using Create React App, please create a custom setup module and reference it using Jest's [`setupFilesAfterEnv`](https://jestjs.io/docs/en/configuration#setupfilesafterenv-array) option in your `jest.config.js`.
 
 <Action>
-  Create a <code>jest.setup.js</code> test teardown file:
+  Create a <code>src/setupTests.js</code> test setup file:
```

Comment:
I'm not sure it's a good idea to recommend a CRA setup as the manual setup choice. There's nothing wrong with `jest.setup.js`, it also being [mentioned in the docs](https://jestjs.io/docs/en/configuration#setupfilesafterenv-array). If you are reading manual setup you are expected to know why you need it, so I don't think the confusion over `jest.config`/`jest.setup` is the case.

Reply:
I didn't feel 100% sure this would make sense to other people. So if you're saying it's not better, I see the point and I reverted the commit.